tigerpath/templates/tigerpath/privacy.html

```diff
@@ -50,10 +51,11 @@ <h4>User Data</h4>
       <div class="text-section">
         <h4>University Data</h4>
         <p>We collect the course listings data from the OIT Web Feed Registry and the Course Offerings webpage, and we compile the requirements information from the Princeton University Undergraduate Announcement website and the official Princeton University department websites.</p>
+        <p>We endeavor to make the displayed course listings and requirements information as accurate as possible. However, errors do occur occasionally, and sometimes the course listings and requirements change before we are able to update them. Therefore, we make no guarantees about the accuracy of our data. The definitive source of data will always come from the university administration and departments; if you are in doubt about the veracity of any university data on our app, you should contact them.</p>
```

Comment:
Looks good, but maybe also add something like:
_... and let us know if you find any errors_
or something to encourage them to notify us of errors.